docs/api.md

```diff
@@ -382,6 +382,61 @@ Endpoints marked as "needs eval access" additionally returns 401 with `error: "U
     };
     ```
 
+## WorksheetMetadata
```

Comment:
I think this should be merged with the previous section since it's still about "worksheets" (in general we put all `/api/x/...` under the same H2)